src/views/pages/property/results-modal.html

```diff
@@ -0,0 +1 @@
+{{> components.modal }}
```

Comment:
I had to create a template where the modal lived since that's how the modal directive works.